src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void updateFruitStorage(FruitTransaction fruitTransaction,
+                                   FruitStorageDao fruitStorageDao) {
+        fruitStorageDao.addFruit(fruitTransaction);
```

Comment:
The `BalanceOperationHandler` should set the balance of a specific fruit to the amount specified in `fruitTransaction`. Currently, it seems to be simply adding the transaction without setting the balance explicitly. Ensure that the balance is being set correctly as per the operation's requirement.